lms/templates/dashboard.html

```diff
@@ -103,12 +103,13 @@ <h2 class="header-courses">${_("My Courses")}</h2>
       </ul>
     % else:
       <section class="empty-dashboard-message">
-        <p>${_("Looks like you haven't enrolled in any courses yet.")}</p>
+        <p>${_("It looks like you aren't yet enrolled in any courses!")}</p>
 
         % if settings.FEATURES.get('COURSES_ARE_BROWSABLE'):
           <a href="${marketing_link('COURSES')}">
-            ${_("Find courses now!")}
+            ${_("Explore edX Courses")}
```

Comment:
@jimabramson and @tasawernawaz - Got approval from @marcotuts for the above as well